crypto/fipsmodule/rand/rand.c

```diff
@@ -1,598 +1,542 @@
-/* Copyright (c) 2014, Google Inc.
```

Comment:
The majorly confusing diff is because I copied all source from `new_rand.c` to `rand.c`. The new implementation now lives in `rand.c` (this file).